examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -1,15 +1,37 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Title: Train an Object Detection Model on Pascal VOC 2007 using KerasCV
+Author: [lukewood](https://github.com/LukeWood), [tanzhenyu](https://github.com/tanzhenyu)
+Date created: 2022/09/27
+Last modified: 2022/12/08
+Description: Use KerasCV to train a RetinaNet on Pascal VOC 2007.
+"""
 import resource
+import sys
 
 import tensorflow as tf
 import tensorflow_datasets as tfds
+from absl import flags
```

Comment:
order wise, the absl should appear before tf.

Reply:
Done.

Reply:
This doesn't seems to be updated.

Reply:
So that seems to be `shell/format.sh` auto-correcting it